plugins/scaffolder/src/components/fields/EntityPicker/EntityPicker.tsx

```diff
@@ -25,14 +25,23 @@ import React, { useCallback, useEffect } from 'react';
 import useAsync from 'react-use/lib/useAsync';
 import { FieldExtensionComponentProps } from '../../../extensions';
 
+/**
+ * The input props that can be specified under `ui:options` for the
+ * `EntityPicker` field extension.
+ *
+ * @public
+ */
 export interface EntityPickerUiOptions {
   allowedKinds?: string[];
   defaultKind?: string;
   allowArbitraryValues?: boolean;
 }
 
 /**
- * Entity Picker
+ * The underling component that is rendered in the form for the `EntityPicker`
```

Comment:
```suggestion
 * The underlying component that is rendered in the form for the `EntityPicker`
```

Signed-off-by: Fredrik Adelöw <freben@gmail.com>